src/index.ts

```diff
@@ -8,3 +8,4 @@ export * from "./types";
 export * from "./utils";
 export { MultiBackend } from "dnd-multi-backend";
 export { HTML5toTouch } from "./HTML5toTouch";
+export { DndProvider } from "react-dnd";
```

Comment:
In v2, `react-dnd` has been changed to peerDependencies. Therefore, you need to install `react-dnd` in the main project that will use this package.

I don't need to export `DndProvider` here, I can just import it directly on the main project side, but what is the advantage of having it exported here?

````
import { DndProvider } from "react-dnd";
```

Reply:
Well, if I only want use this package and no more other using on `react-dnd`, export `DndProvider` from this package can avoid the user mark `react-dnd` as its dependence. Especially when using `pnpm`.

ps: I know less about `peerDependencies`, above just my thought.

Reply:
The `peerDependencies` should not be automatically installed just by installing this package.
So in v2, you will need to install react-dnd separately if you want to use this package. (README will be modified as well.)

```
npm install @minoru/react-dnd-treeview react-dnd
```

In v1, react-dnd was included in this package, but there are many use cases where react-dnd is already installed when this package is used, and such cases cause problems, so in v2, `react-dnd` is not included.

Since react-dnd must be installed, v2 will be used as follows.

```
import { DndProvider } from "react-dnd";
import { Tree, MultiBackend, getBackendOptions } from "@minoru/react-dnd-treeview";

...

return (
  <DndProvider backend={MultiBackend} options={getBackendOptions()}>
    <Tree {...some props} />
  </DndProvider>
);

```

There is no problem to add DndProvider export to src/index.tx. In that case, the import of the package user will be written as follows.

```
import { DndProvider, Tree, MultiBackend, getBackendOptions } from "@minoru/react-dnd-treeview";
```

I am fine with either. I might want to include the `DndProvider` export in the index.ts to allow for both cases. (In that case, no special modification is needed.)